wuvt/defaults.py

```diff
@@ -56,6 +56,7 @@
 ADMINS = []
 MAIL_FROM = "noreply@localhost"
 SMTP_SERVER = "localhost"
+CHART_DEST = "music@wuvt.vt.edu"
```

Comment:
Can we make this "charts@localhost" instead? I would prefer that the default not send email to us.